### ISSUE TITLE
Fix Info Text Color Contrast Issue

### ISSUE THREAD

**Issue by amalyam** [seas-computing/mark-one#215]
### Description

Our `info` text color does not have sufficient color contrast to meet WCAG AA standards when used for normal sized text on a white background (i.e. `background.light`), which is our typical use-case for this color. We want to make this color dark enough to meet WCAG AA standards for sufficient color contrast on a white background.

### To Reproduce

1. Go to the Course Planner [site](https://planning.seas.harvard.edu/courses/courses)
2. Select inspect while highlighting any text functioning as a link, such as the `Courses` tab (alternately, directly find this within an anchor tag in the inspector)
3. Check the color under the styles pane, prior to any changes it is `rgb(78, 136, 199)` or `#4E88C7`
4. Check this value as the foreground option against background `#FFFFFF` in a color contrast checker such as WebAim - it fails at normal text size.

### Expected Behavior

The color for this text needs to pass WCAG AA standards for color contrast against a white background.

### Screenshots

<img width="797" height="506" alt="Image of Course Planner website showing the steps described under To Reproduce, showing the current color for link text" src="https://github.com/user-attachments/assets/942853f0-6173-45af-8878-61e3af16faed" />

<img width="751" height="558" alt="Image of the contrast checker on webaim showing that #4E88C7 as foreground and #FFFFFF as background does not have high enough contrast for normal size text" src="https://github.com/user-attachments/assets/3b31cc89-317c-47bd-90fe-331d343e8daf" />
